src/Home/Home.tsx

```diff
@@ -1,46 +1,112 @@
+import covid from '../assets/covid.png'
 
 
 const Home: React.FC = () => {
-	const [popularCities, setPopularCities] = useState([])
-	const [lowOzoneCities, setOzoneCities] = useState([])
-	const [lowPollutionCities, setPollutionCities] = useState([])
+	const newsCards = [
+  <a
+   href="https://www.cdc.gov/coronavirus/2019-ncov/travelers/travel-during-covid19.html"
+   target="_blank"
+  >
+   <div className="news-card">
+    <img src={covid} alt="COVID virus" />
+    <div className="news-title-overlay">
+     <h4>COVID & US Travel</h4>
+    </div>
+   </div>
+  </a>,
+  <a
+   href="https://www.cdc.gov/nceh/features/wildfires/index.html"
+   target="_blank"
+  >
+   <div className="news-card">
+    <img
+     src={wildfire}
+     alt="forest burning in wildfire"
+    />
+    <div className="news-title-overlay">
+     <h4>Wildfires & Respiration</h4>
+    </div>
+   </div>
+  </a>,
+  <a
+   href="https://www.active.com/outdoors/articles/5-tips-for-successful-altitude-acclimation"
+   target="_blank"
+  >
+   <div className="news-card">
+    <img
+     src={altitude}
+     alt="person hiking in mountains"
+    />
+    <div className="news-title-overlay">
+     <h4>Altitude Acclimation Tips</h4>
+    </div>
+   </div>
+  </a>,
+  <a
+   href="https://travel.usnews.com/rankings/best-beaches-in-the-usa/"
+   target="_blank"
+  >
+   <div className="news-card">
+    <img
+     src={beach}
+     alt="scenic beach under sunny blue skies"
+    />
+    <div className="news-title-overlay">
+     <h4>Best US Beaches</h4>
+    </div>
+   </div>
+  </a>,
+  <a href="https://www.roadtripusa.com/" target="_blank">
+   <div className="news-card">
+    <img
+     src={roadTrip}
+     alt="1970s Volkswagen van and desert landscape"
+    />
+    <div className="news-title-overlay">
+     <h4>USA Road Trip Routes</h4>
+    </div>
+   </div>
+  </a>,
+ ];
 
-	const popularCities = popularCities.map((city: object) => {
-		return (</Card city={props.city}>);
+	const popularCities = popCities.map((city: any) => {
+		return (<Card city={city} />);
 	})
 
-	const ozoneCities = lowOzoneCities.map((city: object) => {
-		return (</Card city={props.city}>);
+	const lowOzoneCities = ozoneCities.map((city: any) => {
+		return (<Card city={city} />);
 	})
 	
-	const pollutionCities = lowPollutionCities.map((city: object) => {
-		return (</Card city={props.city}>);
+	const lowPollutionCities = pollutionCities.map((city: any) => {
+		return (<Card city={city} />);
 	})
 
 	return (
 		<section className="home">
+			<h2 className='carousel-header'>Popular Destinations</h2>
 			<div className='card-carousel'>
 				{popularCities}
 			</div>
+			<h2 className='carousel-header'>Lowest Ozone Pollution</h2>
 			<div className='card-carousel'>
 				{lowOzoneCities}
 			</div>
+			<h2 className='carousel-header'>Lowest Particle Pollution</h2>
 			<div className='card-carousel'>
 				{lowPollutionCities}
 			</div>
+			<h2 className='carousel-header'>Pertinent Readings</h2>
```

Comment:
I love this section! I like that we can give the user optional extra links if they want to read up on more information pertinent to their health. And the cards look AH-mazing!!